jenkins/ec2_bootstrap.sh

```diff
@@ -423,6 +423,24 @@ prepare_cowbuilder() {
     /bin/bash ./bootstrap.sh
 }
 
+install_datadog_agent() {
```

Comment:
You could add 'set -e' at the top of this function rather than use '&&' throughout.